src/librustdoc/html/static/main.js

```diff
@@ -890,6 +890,22 @@
         return "\u2212"; // "\u2212" is '−' minus sign
     }
 
+    $(".search-input").on("keyup",function() {
+        if ($(this).val().length === 0) {
+            window.history.pushState("", "std - Rust", "?search=");
+            $('#main.content').removeClass('hidden');
+            $('#search.content').addClass('hidden');
+        }
+    });
+
+    $('.search-input').on('search', function () {
```

Comment:
Just use a standard event (`input` should do), like [in this PR](https://github.com/rust-lang/rust/pull/28736/files#diff-f3aa05ee448be83e65a501258d3aac2bR728).